### ISSUE TITLE
[Feature Request]Breadcrumb - Add Option to Render as Actual Link with `href="/to"`

### ISSUE THREAD

**Issue by hartmut-co-uk** [iview/iview#3106]
### What problem does this feature solve?
As for SEO and UX in general - it would be nice to have the breadcrumbs `<a />` html elements to be rendered with the actual `href` attribute. Is there a particular reason why this is not the case / could this be added as a property?

So basically for 
```
<Breadcrumb>
  <BreadcrumbItem to="/parent1">parent1</BreadcrumbItem>
  <...>
</Breadcrumb>
```
instead of 
```
<...>
  <a class="ivu-breadcrumb-item-link">parent1</a>
<...>
```
to render as 

```
<...>
  <a href="/parent1" class="ivu-breadcrumb-item-link">parent1</a>
<...>
```

Many Thanks!

### What does the proposed API look like?
Unsure what the best property name may be? 
- `with-href`
- `as-link`
- `href` (as already deprecated - once dropped - bring back as `boolean` field?)

**Comment by SergioCrisostomo** [comment]
I agree that having the native `href` would be nice. If there are backwards compatibility issues at least behind a flag/prop.
Do you want to send a PR suggestion for this so we can discuss it?

**Comment by hartmut-co-uk** [comment]
Sure, will put together in a free minute within the next days - what default behaviour would you suggest? Show href or not? 
I guess as it's a new feature we'd have it `false` by default to not cause a new default rendered output..?
I can see one test - would I need to update documentation somewhere, too?

**Comment by SergioCrisostomo** [comment]
Thank you!

About docs, I think @icarusion updates the docs if the PR lands in the codebase.
About default yes, should be `false` to not break current behaviour.
About prop name, that is always the tricky part 😄 maybe @icarusion has a suggestion? I would say `render-href`. The `href` we should avoid since its a native HTML attribute.

**Comment by icarusion** [comment]
Thanks for feedback @hartmut-co-uk , agree with your idea.
i think `render-href` is good,  or `keep-href`. 
Look forward to your pr, and i will update the doc.
